config/tileset/health.json

```diff
@@ -7,12 +7,14 @@
       "2193": "s3://linz-basemaps/2193/gebco_2020_Nztm2000Quad_305-75m/01F1BFJN8R8P7BXN3XTHC5MT5G",
       "3857": "s3://linz-basemaps/3857/gebco_2020_305-75m/01EDMTM3P563P06TWYQAZRA9F6",
       "name": "gebco_2020_305-75m",
+      "title": "GEBCO Gridded Bathymetry (2020)",
       "maxZoom": 15
     },
     {
       "2193": "s3://linz-basemaps/2193/new_zealand_sentinel_2019-2020_10m/01F6P1G0S78XYXAQB7Y375MZ2Q",
       "3857": "s3://linz-basemaps/3857/new_zealand_sentinel_2019-2020_10m/01ED82TFB1ZE9C75CH5VY3Z92B",
-      "name": "new_zealand_sentinel_2019-2020_10m"
+      "name": "new_zealand_sentinel_2019-2020_10m",
+      "title": "New Zealand Sentinel 10m Imagery (2019-2020)"
```

Comment:
This should be the same title as the one in the `aerial.json`

```
"title": "New Zealand 10m Satellite Imagery (2020-2021)",
"title": "New Zealand 10m Satellite Imagery (2021-2022)",
```

Reply:
That one is different in the imagery name and s3 path. Also this date is older. 
` {
      "2193": "s3://linz-basemaps/2193/nz_satellite_2020-2021_10m_RGB/01FHV9VX322F20JYYZ90ZPPM3R",
      "3857": "s3://linz-basemaps/3857/nz_satellite_2020-2021_10m_RGB/01GQ1HYZYCVRERQQS6FR1P5X7M",
      "name": "nz_satellite_2020-2021_10m_RGB",
      "minZoom": 32,
      "title": "New Zealand 10m Satellite Imagery (2020-2021)",
      "category": "Satellite Imagery"
    },
    {
      "2193": "s3://linz-basemaps/2193/nz_satellite_2021-2022_10m_RGB/01G73E4AMSQ91TXQ3KC90NNPA0",
      "3857": "s3://linz-basemaps/3857/nz_satellite_2021-2022_10m_RGB/01GQ1J1V05X64SDTM673WTMJ7J",
      "name": "nz_satellite_2021-2022_10m_RGB",
      "title": "New Zealand 10m Satellite Imagery (2021-2022)",
      "category": "Satellite Imagery"
    },`